crates/viewer/re_time_panel/Cargo.toml

```diff
@@ -25,10 +25,11 @@ re_data_ui.workspace = true
 re_entity_db.workspace = true
 re_format.workspace = true
 re_int_histogram.workspace = true
-re_log_types.workspace = true
 re_log.workspace = true
+re_log_types.workspace = true
```

Comment:
somewhat curious this needs log types actually and not just types_core, ah well...

Reply:
It was there already, was just reordered.

``` rust
use re_log_types::{
    ApplicationId, ComponentPath, EntityPath, ResolvedTimeRange, TimeInt, TimeReal, TimeType,
};
```